Manito/Manito/Screens/Main/MainViewController.swift

```diff
@@ -27,20 +27,24 @@ class MainViewController: BaseViewController {
     
     // MARK: - property
     
-    private var appTitleView = AppTitleView ()
+    private var appTitleView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = ImageLiterals.imgLogo
+        return imageView
+    }()
     
     private let settingButton = SettingButton()
     
-    private var lightImage: UIImageView = {
-        let image = UIImageView()
-        image.backgroundColor = .red
-        return image
+    private var imgStar: UIImageView = {
```

Comment:
p5;

```suggestion
    private var imgStar = UIImageView(image: ImageLiterals.imgStar)
```
이렇게 할 수 있어요!

Reply:
디너 최고^__^

Reply:
수정했습니다.